js/packages/web/src/components/Analytics/index.tsx

```diff
@@ -80,35 +80,37 @@ export function AnalyticsProvider(props: { children: React.ReactNode }) {
 
   function pageview(path: string) {
     if (!gtag) return;
-    gtag('event', 'page_view', {
-      page_location: window.location.href, // important to overwrite to keep fragments
-      page_path: path, // React router provides the # route as a regular path
+    track('page_view', {
```

Comment:
Makes sense to keep the pageview function for now as it is usually viewed as a separate event than tracking. It's just that for GA4 it's essentially the same (when it comes to virtual pageviews)